.gitignore

```diff
@@ -26,3 +34,5 @@ ccdaservice/node_modules
 
 # testing
 .phpunit.result.cache
+.idea/workspace.xml
+.idea/workspace.xml
```

Comment:
the `/.idea/*` above should capture any related files so would remove these as well